bin/check-package-locks.js

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * This is an internal script to verify that local monorepo dependencies
+ * are excluded from package-lock files.
+ */
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+const promisify = require('util').promisify;
+
+const readFile = promisify(fs.readFile);
+
+const Project = require('@lerna/project');
+
+async function checkPackageLocks() {
+  const project = new Project(process.cwd());
+  const packages = await project.getPackages();
+  const rootPath = project.rootPath;
+  const lockFiles = packages.map(p =>
+    path.relative(rootPath, path.join(p.location, 'package-lock.json')),
+  );
+
+  const checkResults = await Promise.all(
+    lockFiles.map(async lockFile => {
+      return {lockFile, violations: await checkLockFile(lockFile)};
+    }),
+  );
+  const badPackages = checkResults.filter(r => r.violations.length > 0);
+  if (!badPackages.length) return 0;
+
+  console.error();
```

Comment:
Can we not use `\n` for new lines with console.error (I'm assuming that's why we have the two calls before and after the `invalid` lock file message)?

Reply:
Are you asking me to use `\n` instead of calling `console.error()`?